src/inmanta/protocol/rest/__init__.py

```diff
@@ -631,7 +630,7 @@ async def _execute_call(
                 arguments.auth_username if arguments.auth_username else "<>",
             )
 
-            with logfire.span("Calling method " + config.method_name, **arguments.call_args):
```

Comment:
I really don't like this kwargs at the end, because they can allow injection of arguments and are impossible to type.